ts/Series/HLC/HLCSeries.ts

```diff
@@ -0,0 +1,485 @@
+                            (point as any)[translated[i]] =
+                                yAxis.toPixels(value, true);
+                        }
+                    }
+                );
+
+            // Align the tooltip to the high value to avoid covering the
+            // point
+            (point.tooltipPos as any)[1] =
+                (point.plotHigh as any) + yAxis.pos - series.chart.plotTop;
+        });
+    }
+
+    /* eslint-enable valid-jsdoc */
+
+}
+
+/* *
+ *
+ *  Prototype Properties
+ *
+ * */
+
+interface HLCSeries {
+    pointAttrToOptions: Record<string, string>;
+    pointClass: typeof HLCPoint;
+    init(): void;
+    toYData(point: HLCPoint): Array<number>;
+}
+extend(HLCSeries.prototype, {
+    animate: null as any, // Disable animation
+    directTouch: false,
+    pointArrayMap: ['high', 'low', 'close'],
+    pointAttrToOptions: {
+        stroke: 'color',
+        'stroke-width': 'lineWidth'
+    },
+    pointValKey: 'close'
+});
+
+HLCSeries.prototype.pointClass = HLCPoint;
+
+/* *
+ *
+ *  Registry
+ *
+ * */
+
+declare module '../../Core/Series/SeriesType' {
+    interface SeriesTypeRegistry {
+        hlc: typeof HLCSeries;
+    }
+}
+SeriesRegistry.registerSeriesType('hlc', HLCSeries);
+
+/* *
+ *
+ *  Default Export
+ *
+ * */
+
+export default HLCSeries;
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * A `hlc` series. If the [type](#series.hlc.type) option is not
+ * specified, it is inherited from [chart.type](#chart.type).
+ *
+ * @extends   series,plotOptions.hlc
+ * @excluding dataParser, dataURL
+ * @product   highstock
+ * @apioption series.hlc
+ */
+
+/**
+ * An array of data points for the series. For the `hlc` series type,
+ * points can be given in the following ways:
+ *
+ * 1. An array of arrays with 4 or 3 values. In this case, the values correspond
+ *    to `x,high,low,close`. If the first value is a string, it is applied
+ *    as the name of the point, and the `x` value is inferred. The `x` value can
+ *    also be omitted, in which case the inner arrays should be of length 4\.
+ *    Then the `x` value is automatically calculated, either starting at 0 and
+ *    incremented by 1, or from `pointStart` and `pointInterval` given in the
+ *    series options.
+ *    ```js
+ *    data: [
+ *        [0, 5, 6, 7],
+ *        [1, 4, 8, 2],
+ *        [2, 3, 4, 10]
+ *    ]
+ *    ```
+ *
+ * 2. An array of objects with named values. The following snippet shows only a
+ *    few settings, see the complete options set below. If the total number of
+ *    data points exceeds the series'
+ *    [turboThreshold](#series.hlc.turboThreshold), this option is not
+ *    available.
+ *    ```js
+ *    data: [{
+ *        x: 1,
+ *        high: 4,
+ *        low: 5,
+ *        close: 2,
+ *        name: "Point2",
+ *        color: "#00FF00"
+ *    }, {
+ *        x: 1,
+ *        high: 3,
+ *        low: 6,
+ *        close: 7,
+ *        name: "Point1",
+ *        color: "#FF00FF"
+ *    }]
+ *    ```
```

Comment:
Please add the third option where you describe how to use HLC series with OHLC data.

Reply:
That would suggest only HLC can work that way (`useOhlcData` option is available to all series)